package.json

```diff
@@ -60,6 +61,7 @@
     "simple-git": "^1.65.0",
     "sinon": "^2.3.2",
     "style-loader": "^0.19.1",
+    "typescript": "^2.9.0-dev.20180325",
```

Comment:
maybe its better to start with stable release?

Reply:
Fair point. With this work I've seen some bugs in TypeScript itself. Hoping they're fixed in the nightly builds. We should not go to master with a nightly build :)